src/main/resources/assets/luminiferous_uplands/lang/en_us.json

```diff
@@ -10,6 +10,7 @@
   "block.luminiferous_uplands.uplander_stone_brick_stairs": "Uplander Stone Brick Stairs",
   "block.luminiferous_uplands.uplander_stone_brick_slab": "Uplander Stone Brick Slab",
   "block.luminiferous_uplands.uplander_stone_brick_wall": "Uplander Stone Brick Wall",
+  "block.luminiferous_uplands.unbreakable_uplander_stone_bricks": "Unbreakable Uplander Stone Bricks",
```

Comment:
Any way to generalize this and have it use the translated name of the block it wraps, and use `Unbreakable %s` type thing?

Reply:
Possible, but it's not as friendly for localization. Vanilla spawn eggs used to use generated localized names, but now they all have separate names.